apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1459,8 +1478,62 @@ function validateCourseInstance({
     }
   }
 
+  if (courseInstance.selfEnrollment.useEnrollmentCode !== false) {
+    warnings.push('"selfEnrollment.useEnrollmentCode" is not configurable yet.');
+  }
```

Comment:
👀 was this supposed to have been added before? Do we want a TODO like with the below warning?

Reply:
This shouldn't be on this branch.